packages/lesswrong/components/localGroups/CommunityHome.tsx

```diff
@@ -163,6 +168,15 @@ const CommunityHome = ({classes}: {
                 </PostsList2>}
               </AnalyticsContext>
             </SingleColumnSection>
+            
+            <SingleColumnSection>
+              <SectionTitle title="Online Groups">
+                {canCreateEvents && <GroupFormLink />}
```

Comment:
Minor consideration, but you could auto-populate the "This is an online group" checkbox by passing down an `isOnline` prop through GroupFormLink into GroupFormDialog, which then sets `prefilledProps={{isOnline}}}` on WrappedSmartForm. Not a huge deal either way.

Reply:
I like this idea! Added :)